spec/services/file_identifier_service_spec.rb

```diff
@@ -56,8 +56,8 @@
 
       let(:status) { instance_double(Process::Status, success?: true) }
 
-      it 'returns mimetype' do
-        expect(mimetype).to eq('text/plain')
+      it 'returns pronom id and mimetype' do
+        expect(mimetype).to eq(['x-fmt/111', 'text/plain'])
```

Comment:
how about changing the name of the subject from `mimetype` to something more descriptive now?

Reply:
Done.